kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/ProcurementCardLevel3Transport.java

```diff
@@ -0,0 +1,144 @@
+package edu.arizona.kfs.fp.businessobject;
+
+import java.sql.Date;
+import java.util.LinkedHashMap;
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.krad.bo.PersistableBusinessObjectBase;
+
+public class ProcurementCardLevel3Transport extends PersistableBusinessObjectBase {
+
+	private String documentNumber;
+	private Integer financialDocumentTransactionLineNumber;
+	private String passengerName;
+	private Date departDate;
+	private String departureCity;
+	private String agencyCode;
+	private String agencyName;
+	private long ticketNumber;
```

Comment:
This appears to be a numeric field per the JPMC spec. Originally this property was an Integer but was changed to long in the following commit: https://github.com/ua-eas/eas-ksd-kfs-3.0/commit/ee22b6a8d8cfe7e7214a3608e2df7b09d3ecfbe1#diff-4dd3a0c0b2bacda77642964478df76c8. The Jira is KITT-435, but I could not find any comments giving the reason for the change.